tests/test_converter.py

```diff
@@ -46,6 +46,10 @@ class SerializableEnum(IntEnum):
     FOO = 1
 
 
+class SerializableStrEnum(StrEnum):
```

Comment:
We run tests all the way back to Python 3.7, so you'll need to make the tests conditional on 3.11 too

Reply:
Updated the test and confirmed both the 3.7 and 3.11 Python versions passes the test on my local machine. I should've checked it in the first place ^^; Please check the change :pray: